packages/venia-concept/jest.config.js

```diff
@@ -0,0 +1,34 @@
+const { join } = require('path');
+
```

Comment:
This is an old PR; Jest is configured in the root now, so this whole file needs to be deleted.

Reply:
Yep, I already deleted it just haven't pushed.